torch/csrc/jit/codegen/cuda/ir_nodes.cpp

```diff
@@ -1027,7 +1027,6 @@ Allocate::Allocate(Val* _val, Val* _size)
         ".");
   }
   addInput(_size);
-  addInput(_val);
```

Comment:
Why?

Reply:
Naoya found an issue with `fusion->printKernel` and the cache functions. It was adding an extra output to the TensorView.

Reply:
thanks